solutions/pytest_folder/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from selenium import webdriver
+import json
```

Comment:
Сортировка импортов плюс https://github.com/zinka22/Selenium-cource/pull/39#discussion_r1639467712.

```suggestion
import json
from pathlib import Path

import pytest
from selenium import webdriver
```